src/main/java/com/minecolonies/coremod/colony/buildings/BuildingDeliveryman.java

```diff
@@ -69,6 +75,19 @@ public int getMaxBuildingLevel()
         return 5;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS109)